cmd/reactroles/helm/templates/configmap.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{include "reactroles.fullname" .}}
+data:
+  DISCORD_ROLECHANNEL: {{.Values.reactroles.discord.role.channel}}
+  DISCORD_ROLEMESSAGE: {{.Values.reactroles.discord.role.message}}
+  DISCORD_ROLEADDROLEID: {{.Values.reactroles.discord.role.addRoleId}}
+  DISCORD_ROLEREMOVEROLEID: {{.Values.reactroles.discord.role.removeRoleId}}
+  DISCORD_TOKEN: {{.Values.reactroles.discord.token}}
+
+  POSTGRES_DB: {{.Values.reactroles.postgres.db}}
+  POSTGRES_HOST: {{.Values.reactroles.postgres.host}}
+  POSTGRES_PASSWORD: {{.Values.reactroles.postgres.password}}
+  POSTGRES_PORT: {{.Values.reactroles.postgres.port}}
+  POSTGRES_SSL: {{.Values.reactroles.postgres.ssl}}
+  POSTGRES_TIMEZONE: {{.Values.reactroles.postgres.timezone}}
+  POSTGRES_USER: {{.Values.reactroles.postgres.user}}
```

Comment:
This can be moved into the Values by replacing it with
```
  {{- range $key, $val := .Values.reactroles.discord }}
  {{ $key }}: {{ $val | quote }}
  {{- end }}
  {{- range $key, $val := .Values.reactroles.postgres }}
  {{ $key }}: {{ $val | quote }}
  {{- end }}
```

slightly easier to migrate between different projects and lets you only modify one location when you're updating the application further down the road.